src/m365/base/ContextCommand.spec.ts

```diff
@@ -0,0 +1,161 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../appInsights';
+import { Logger } from '../../cli/Logger';
+import { CommandError } from '../../Command';
+import ContextCommand from './ContextCommand';
+import { sinonUtil } from '../../utils/sinonUtil';
+import * as fs from 'fs';
+import { Hash } from '../../utils/types';
+
+class MockCommand extends ContextCommand {
+  public get name(): string {
+    return 'mock';
+  }
+
+  public get description(): string {
+    return 'Mock command';
+  }
+
+  public mockSaveContextInfo(contextInfo: Hash, logger: Logger) {
+    this.saveContextInfo(contextInfo, logger);
+  }
+
+  public async commandAction(): Promise<void> {
+  }
+
+  public commandHelp(): void {
+  }
+}
+
+describe('ContextCommand', () => {
+  let cmd: MockCommand;
+  let log: any[];
+  let logger: Logger;
+  const contextInfo: Hash = {};
+  let loggerLogSpy: sinon.SinonSpy;
+
+
+  before(() => {
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
```

Comment:
Hi, just popping in to notify you that we should update this line to:

```
sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
```

to align it with our latest change: #4215